src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Helpers/DecisionHelper.cs

```diff
@@ -108,15 +118,12 @@ private static XacmlJsonCategory CreateResourceCategory(string org, string app,
             XacmlJsonCategory resourceCategory = new XacmlJsonCategory();
             resourceCategory.Attribute = new List<XacmlJsonAttribute>();
 
-            if (string.IsNullOrWhiteSpace(instanceOwnerPartyId))
-            {
-                resourceCategory.Attribute.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.InstanceId, instanceGuid, DefaultType, DefaultIssuer, includeResult));
-            }
-            else if (string.IsNullOrWhiteSpace(instanceGuid))
+            if (!string.IsNullOrWhiteSpace(instanceOwnerPartyId))
             {
                 resourceCategory.Attribute.Add(CreateXacmlJsonAttribute(AltinnXacmlUrns.PartyId, instanceOwnerPartyId, DefaultType, DefaultIssuer, includeResult));
```

Comment:
Just a suggestion. Change the types of the parameter to this method. Convert to string at the last possible time.

Reply:
keeping the string input as it would result in having to convert from string to int back to string in other functions